MusicStore.Business/Services/AdminService.cs

```diff
@@ -44,5 +44,34 @@ public IList<UserAccount> GetFullListOfUsers()
 
             return usersAccounts.Select(_mapUser.AutoMap).ToList();
         }
+
+        public bool EditUserAccount(UserAccount userDomain)
+        {
+            if (userDomain == null)
+            {
+                throw new ArgumentNullException("userDomain", "Can not update user, because it is null");
+            }
+
+            var userDataAccess = _userRepository.GetItem(userDomain.Id);
```

Comment:
Check id to have any value. This will reduce database calls

Reply:
+